packages/ui-position/src/Position/index.js

```diff
@@ -327,27 +295,20 @@ class Position extends Component {
   }
 
   renderContent() {
-    let content = ComponentIdentifier.pick(
-      Position.Content,
-      this.props.children
-    )
-    if (!content) {
-      content = ensureSingleChild(this.props.children)
-    }
+    let content = ensureSingleChild(this.props.children)
 
     if (content) {
       content = safeCloneElement(content, {
         ref: (el) => {
           this._content = el
         },
         style: {
+          boxSizing: 'border-box',
+          zIndex: this.props.styles.zIndex,
```

Comment:
This is a new breaking change, sadly it is the only way I found to put these 2 css props on the component, emotion doesn't work like this. Also, kind of funny, because if the content block is a View or ContextView, it filters out most of the styles anyway, but the live version works like this too.  Added breaking change comment on the commit.